plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -103,13 +110,25 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
           this.logger.warn(`failed to add user to group: ${e.message}`);
         }
       });
-
+      const profileAnnotations: Record<string, string> = {};
+      if (this.customAttributesToAnnotationWhitelist.length) {
+        for (const [key, value] of new Map(Object.entries(group.profile))) {
+          const stringKey = key.toString();
+          if (this.customAttributesToAnnotationWhitelist.includes(stringKey)) {
+            profileAnnotations[stringKey] = value.toString();
```

Comment:
Could you please namespace the attribute with something ? For example "okta.com/<stringKey>"

Reply:
mmm wouldn't organisations benefit from having their own namespacing there? in the end the sense of this is to be able to have further custom integrations 🤔  i'm open to discuss it though